.github/workflows/ci.yml

```diff
@@ -39,6 +41,10 @@ env:
 # The MSRV jobs run only cargo check because different clippy versions can disagree on goals and
 # running tests introduces dev dependencies which may require a higher MSRV than the bare package.
 #
+# For no_std checks we target x86_64-unknown-none, because this target doesn't support std
```

Comment:
(there are reasonable arguments to use `wasm32v1-none` at some point, but not for this PR - IIRC, it was only stabilised in 1.84. Cc @xStrom)